packages/vscode-docker-registries/src/clients/GitHub/GitHubRegistryDataProvider.ts

```diff
@@ -117,6 +124,32 @@ export class GitHubRegistryDataProvider extends RegistryV2DataProvider {
         return this.authenticationProvider;
     }
 
+    protected override async getTagDetails(repository: V2Repository, tag: string): Promise<Date | undefined> {
+        const tagDetailResponse = await registryV2Request<Blob>({

```

Comment:
Does the original strategy not work with GitHub? I would have expected it to since it's a fully V2-compliant registry

Reply:
Does the strategy used for GitHub work for generic V2 registries?

Reply:
i haven't tried that, let me look

Reply:
update: no it does not

Reply:
That's a bummer 😞